dev-guide.md

```diff
@@ -56,23 +56,20 @@ Our docker setup reads environmental variables from .env as described down the p
 
 ##### Setting up a Python environment
 
-We recommend setting up an isolated virtual environment where you install the dependencies. You can set one up by following [this guide](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment). Here are the steps for Ubuntu:
-
+Start by installing [pipenv](https://pipenv.pypa.io/en/latest/install/#installing-pipenv)
+On Ubuntu, you can accomplish this by running:
 ```
-# install Python 3, venv, and Postgres packages
-sudo apt-get install python3-dev python3-venv libpq-dev
-# create a new virtual environment
-python3 -m venv venv_smallboard
+$ sudo apt install pipenv
 ```
-
-Once you've set up the new virtual environment, activate it and install Small Board's dependencies:
-
+Once you have pipenv, you can create an environment by running:
 ```
-source venv_smallboard/bin/activate
-(venv_smallboard)$ pip install -r requirements.txt
+pipenv install --dev
+```
+This creates a virtual environment and installs the dependencies and dev dependencies
+You can then activate the virtual environment by running
+```
+pipenv shell
```

Comment:
We should update the docs below about calling `./manage.py ...`

(you already had plans for this, but commenting just for the record)

Reply:
is this actually different from running whatever command is already in the docs (provided one is inside the env/shell)?